app/build.gradle

```diff
@@ -27,6 +27,10 @@ android {
             applicationIdSuffix ".debug"
             versionNameSuffix "-debug"
         }
+        qa_test {
```

Comment:
@gscdev Alternatively, we could name this build type `staging` instead of `qa_test `, producing this correspondence with the TEST/PROD servers:
```
debug → TEST
qa_test → TEST
release → PROD
```

What do you think?

Reply:
I think it's okay. I will set the same name in the launcher elimu-ai/launcher#57